tests/virt_autotest/login_console.pm

```diff
@@ -240,6 +241,11 @@ sub login_to_console {
         assert_script_run('getconf PAGESIZE');
     }
 
+    # Start modular libvirtd services on system bootup for SLE Micro 6.0 onwards.
+    if (is_sle_micro('>=6.0')) {
```

Comment:
As explained in an earlier PR, this is NOT the recommended conditional soft-fail. The condition is to verify if the problem that the bug describes happens, then with this if being true, claim record_soft_fail. To be specifc,
the condition is sth like this:

if (check modular libvirt service, and they are disable&inactive) {
   record_soft_failure("bsc#1218093 ....")
   restart_libvirtd;
}